test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -82,6 +81,8 @@ func (p paymentSender) sendPayment(a *require.Assertions, f *fixtures.RestClient
 	a.NoError(err)
 }
 
+const roundTimeForStateproofTests = 3 * time.Minute
```

Comment:
Name a bit confusing since it's not 3 minute round time, but rather a 3 minute timeout